cmd/controller-manager/app/controller-manager.go

```diff
@@ -89,6 +89,7 @@ member clusters and does the necessary reconciliation`,
 	cmd.Flags().StringVar(&federationConfig, "federation-config", "", "Path to a federation config yaml file. Test only.")
 	cmd.Flags().StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	cmd.Flags().StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	cmd.Flags().StringVar(&defaultScope, "default-scope", "Cluster", "controls the default scope if the FederationConfig is not supplied.")
```

Comment:
- Please add a TODO comment above the flag definition as follows:

```golang
// TODO(sohankunkerkar) Remove when no longer necessary.  This flag is a temporary addition to support Red Hat's downstream testing efforts.  Its continued existence should not be relied upon.
```

- The first word in the help text should be capitalized, as per the examples of the help text of other flags.  Consistency is very important for collaborative development, and I encourage you to do your best to ensure that changes you make are in keeping with the example of related code.

- Please reuse constants when the exist.  In this case, `string(apiextv1b1.ClusterScoped)` is preferable to `"Cluster"`.